content/03.ml-concepts.md

```diff
@@ -1,24 +1,28 @@
 ## Tip 1: Concepts that apply to machine learning also apply to deep learning {#concepts}
-Deep learning is a distinct subfield of machine learning, but it is still a subfield.
-DL has proven to be an extremely powerful paradigm capable of outperforming “traditional” machine learning approaches in certain contexts, but it is not immune to the many limitations inherent to machine learning.
-Many best practices for machine learning also apply to deep learning.
-Like all computational methods, deep learning should be applied in a systematic manner that is reproducible and rigorously tested.
 
-Those developing deep learning models should select data that are relevant to the problem at hand; non-salient data can hamper performance or lead to spurious conclusions.
-Biases in testing data can also unduly influence measures of model performance, and it may be difficult to directly identify confounders from the model.
-Investigators should consider the extent to which the outcome of interest is likely to be predictable from the input data and begin by throughly inspecting the input data.
-Suppose that there are robust heritability estimates for a phenotype that suggest that the genetic contribution is modest but a deep learning model predicts the phenotype with very high accuracy.
+While deep learning has inarguably surpassed general machine learning for artificial intelligence and predictive modeling applications in terms of public awareness, deep learning still remains a subfield of machine learning.
+DL has proven to be an extremely powerful paradigm capable of outperforming "traditional" or "conventional" machine learning approaches. This is partly owed to the fact that DL algorithms are less dependent on manual feature extraction and are capable of achieving high predictive performance when operating on raw-data, such as images and text. directly.
```

Comment:
I think we should retain the caveat about "certain contexts", since there are still places where deep learning doesn't make sense (e.g. small sample size problems are probably still best approached with strictly Bayesian methods). 

This also needs to be split into one line per sentence.

Reply:
I think the issue should have disappeared after rephrasing. I think the main point we want to get at here is that as DL models usually have a larger capacity and on larger feature sets (usually raw data instead of manually curated features), using machine learning best practices for model evaluation is ever so important (we list various caveats). Regarding which approach is better for a given dataset etc, I think that's covered in different tips, incl. the tip on baselines and the one on overfitting.